machine/machine_core/ssh_connection.py

```diff
@@ -69,6 +69,22 @@ def message(self, *args):
         if self.verbose:
             sys.stderr.write(" ".join(args) + '\n')
 
+    def print_console_log(self):
+        """Prints VM's console to stderr"""
+        console_file = "/tmp/console.log"
```

Comment:
This looks wrong. Is this a test leftover? If this path is set anywhere, it won't work on our bots, as they run many VMs in parallel. Also, fixed paths in /tmp are a no-go.

Reply:
Fixed